### PR TITLE
Controller

### PR DESCRIPTION
**Pull request by barakadanny**
In this PR, The following are the things worked on:
- Use controllers to handle requests and render empty views.
- Use params from browser requests in a safe way.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div>
+  <h1 class="font-bold text-4xl">Users#index</h1>
+  <p>Find me in app/views/users/index.html.erb</p>
+</div>
```
Comment (FrederickMih) [inline app/views/users/index.html.erb:4]: - [ ] Great job for adding the view file for the #index action :+1: @barakadanny, can you remember you equally added the #show action for the users_controller? Let's equally add a view file for the #show action as requested in the requirement that's in ` #app/views/users/show.html.erb`

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div>
+  <h1 class="font-bold text-4xl">Posts#index</h1>
+  <p>Find me in app/views/posts/index.html.erb</p>
+</div>
```
Comment (FrederickMih) [inline app/views/posts/index.html.erb:4]: - [ ] As pointed above :arrow_up: , you'll equally require a view for the #show action.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @barakadanny, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Unnecessary routes were not exposed :+1: 
- PR is descriptive :+1: 
- All Linter checks passed :heavy_check_mark: 


## Required Changes ♻️
N/A
_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (barakadanny) [reply app/views/posts/index.html.erb:4]: @FrederickMih Thank you for the review.
The changes have been made.

**Review by mukhammadrizooff — APPROVED** [review]
Hi @barakadanny, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**